helm/pay-wallet-values-dev.yaml

```diff
@@ -76,8 +76,12 @@ microservice-chart:
     OTEL_METRICS_EXPORTER: otlp
     OTEL_LOGS_EXPORTER: otlp
     OTEL_TRACES_SAMPLER: "always_on"
+    EXPIRATION_QUEUE_NAME: pagopa-d-itn-pay-wallet-cdc-queue
+    EXPIRATION_QUEUE_TTL_SECONDS: "3600"
```

Comment:
sec vs seconds suffix
In order to keep the code readable, please choose only one of the two suffixes

Reply:
i'll fix it. no problem
i didnt think it was an issue as this is the format used in wallet-service